modules/install-consul/install-consul

```diff
@@ -82,11 +82,32 @@ function install_supervisord_amazon_linux {
   sudo pip install supervisor
 
   # On Amazon Linux, /usr/local/bin is not in PATH for the root user, so we add symlinks to /usr/bin, which is in PATH
-  if [[ ! -f "/usr/bin/supervisorctl" ]]; then
-    sudo ln -s /usr/local/bin/supervisorctl /usr/bin/supervisorctl
+  if [[ ! -f "/usr/local/bin/supervisorctl" ]]; then
+    if [[ -f "/usr/bin/supervisorctl" ]]; then
+      sudo ln -s /usr/bin/supervisorctl /usr/local/bin/supervisorctl
+    else
+      echo "Unable to find supervisorctl tool... exiting"
+      exit -5
+    fi
+  elif [[ ! -f "/usr/bin/supervisorctl" ]]; then
+    # IF supervisorctl does NOT exist in /usr/bin, check for it in /usr/local/bin and link it to /usr/bin
+    if [[ -f "/usr/local/bin/supervisorctl" ]]; then
+      sudo ln -s /usr/local/bin/supervisorctl /usr/bin/supervisorctl
+
+    fi
   fi
-  if [[ ! -f "/usr/bin/supervisord" ]]; then
-    sudo ln -s /usr/local/bin/supervisord /usr/bin/supervisord
+
+  if [[ ! -f "/usr/local/bin/supervisord" ]]; then
+    if [[ -f "/usr/bin/supervisord" ]]; then
+      sudo ln -s /usr/bin/supervisorctl /usr/local/bin/supervisorctl
+    else
+      echo "Unable to find supervisord... exiting"
+      exit -5
+    fi
+  elif [[ ! -f "/usr/bin/supervisord" ]]; then
+    if [[ -f "/usr/local/bin/supervisord" ]]; then
+      sudo ln -s /usr/local/bin/supervisorctl /usr/bin/supervisorctl
```

Comment:
Same comment here

Reply:
Bleh good catch!  Really should have abstracted it to a function.  Will fix :) I'll blame baby brain!